src/annotator/integrations/test/html-test.js

```diff
@@ -0,0 +1,69 @@
+import { HTMLIntegration, $imports } from '../html';
+
+describe('HTMLIntegration', () => {
+  let fakeHtmlAnchoring;
+  let fakeHtmlMetadata;
+
+  beforeEach(() => {
+    fakeHtmlAnchoring = {
+      anchor: sinon.stub(),
+      describe: sinon.stub(),
+    };
+
+    fakeHtmlMetadata = {
+      getDocumentMetadata: sinon.stub().returns({ title: 'Example site' }),
+      uri: sinon.stub().returns('https://example.com/'),
+    };
+
+    const HTMLMetadata = sinon.stub().returns(fakeHtmlMetadata);
+    $imports.$mock({
+      '../anchoring/html': fakeHtmlAnchoring,
+      './html-metadata': { HTMLMetadata },
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  it('implements `anchor` and `destroy` using HTML anchoring', () => {
+    const integration = new HTMLIntegration();
+    assert.equal(integration.anchor, fakeHtmlAnchoring.anchor);
+    assert.equal(integration.describe, fakeHtmlAnchoring.describe);
+  });
+
+  describe('#contentContainer', () => {
+    it('returns body by default', () => {
+      const integration = new HTMLIntegration();
+      assert.equal(integration.contentContainer(), document.body);
+    });
+  });
+
+  describe('#destroy', () => {
+    it('cleans up integration', () => {
+      new HTMLIntegration().destroy();
+    });
+  });
```

Comment:
This test and the one below are stubs just to check that the methods don't result in any errors. There are no expected effects and hence no asserts.